hydromt_wflow/workflows/soilgrids.py

```diff
@@ -841,3 +846,94 @@ def soilgrids_sediment(ds, ds_like, usleK_method, logger=logger):
         ds_out[var].raster.set_nodata(nodata)
 
     return ds_out
+
+
+def update_soil_with_paddy(
+    ds: xr.Dataset,
+    ds_like: xr.Dataset,
+    paddy_mask: xr.DataArray,
+    soil_fn: str = "soilgrids",
+    update_c: bool = True,
+    wflow_layers: List[int] = [50, 100, 50, 200, 800],
+    target_conductivity: List[Union[None, int, float]] = [None, None, 5, None, None],
+    logger=logger,
+):
+    """
+    Update c and kvfrac soil properties for paddy fields.
+
+    Parameters
+    ----------
+    ds : xarray.Dataset
+        Dataset containing soil properties.
+    ds_like : xarray.DataArray
+        Dataset at model resolution.
+        Required variables: wflow_subcatch, KsatVer, f
+    paddy_mask : xarray.DataArray
+        Dataset containing paddy fields mask.
+    soil_fn : str
+        soilgrids version {'soilgrids', 'soilgrids_2020'}
+    update_c : bool
+        Update c based on change in wflow_layers.
+    wflow_layers : list
+        List of soil layer depths [cm] for which c is calculated.
+    target_conductivity : list
+        Target conductivity for each wflow layer.
+
+    Returns
+    -------
+    soil_out : xarray.Dataset
+        Dataset containing updated soil properties.
+    """
+    if len(wflow_layers) != len(target_conductivity):
+        raise ValueError(
+            "Lengths of wflow_thicknesslayers and target_conductivity does not " "match"
```

Comment:
Small "typo" probably caused by formatting
```suggestion
            "Lengths of wflow_thicknesslayers and target_conductivity does not match"
```